_layouts/posts.html

```diff
@@ -27,15 +27,29 @@
                     <a href="{{ post.url }}"><p>{{ post.title | escape }}</p></a>
                 {% endif %}
                 </div>
-
-                <p class="subtitle">
+                
+                    <p class="subtitle">
                     <time datetime="{{ post.date | date: date_format }}">{{ post.date | date: date_format }}</time>
                     - {{ authors | remove_first: ', ' | strip }}
-                </p>
+                    </p> 
+                
+        <div class="post-categories">
+            {% if post %}
+              {% assign categories = post.categories %}
+            {% endif %}
+
+            {% for category in categories %}
+            <button class="catag"><a href="{{site.url}}/categories/#{{category|slugize}}" style="color: white; font-style: italic; font-size: 16px; font-family: sans-serif;">{{category | join: ', '}}</a></button>
```

Comment:
- Can you move the style to the CSS file? 
- Can the class name be something like `button-category`?
- Why do you need the join there?